Makefile

```diff
@@ -96,3 +96,8 @@ rm-docker-postgres: stop-docker-postgres
 .PHONY: check-file-headers
 check-file-headers:
 	git ls-files | node lib/bin/check-file-headers.js
+
+.PHONY: api-docs
+api-docs:
+	(test "$(docker images -q odk-docs)" || docker build --file odk-docs.dockerfile -t odk-docs .) && \
```

Comment:
I feel like I usually see `test` with an operator like `-n`. What does it do when it's just passed a string?

Reply:
without any option `test` utility does this: `string        True if string is not the null string.`